Fix intermittent failing IT LATEST_RELEASE[7.9]_vs2017-win2016

### Description

The `vs2017_latest` test (vs2017-win2016) is failing quite often, intermittently with a timeout.

### Repro steps

It's intermittent.

If possible then please create a git repository with a repro sample or attach a zip to the issue.

### Expected behavior

Should not fail.

### Actual behavior

[Example](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=40092&view=logs&j=3dc0cc2f-df27-50a4-aa47-88f24259eefd&t=f11e65dd-4e97-58ba-a53b-edb75b763a81&s=8d806912-ebd6-57b6-8e60-1c9bc1968d80) ([raw logs](https://dev.azure.com/sonarsource/399fb241-ecc7-4802-8697-dcdd01fbb832/_apis/build/builds/40092/logs/104))

>2021-11-26T14:14:16.3724040Z [ERROR] Tests run: 34, Failures: 0, Errors: 1, Skipped: 6, Time elapsed: 644.387 s <<< FAILURE! - in com.sonar.it.scanner.SonarScannerTestSuite
2021-11-26T14:14:16.3725947Z [ERROR] testProjectTypeDetectionWithWrongCasingReferenceName(com.sonar.it.scanner.msbuild.ScannerMSBuildTest)  Time elapsed: 80.903 s  <<< ERROR!
2021-11-26T14:14:16.3727303Z com.sonar.orchestrator.util.CommandException: Timeout exceeded: 60000 ms [command: cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe" "/t:Restore,Rebuild" DotnetProjectTypeDetection.sln"]
2021-11-26T14:14:16.3728455Z 	at com.sonar.orchestrator.util.CommandExecutor.execute(CommandExecutor.java:98)
2021-11-26T14:14:16.3729324Z 	at com.sonar.it.scanner.msbuild.TestUtils.runMSBuildQuietly(TestUtils.java:263)
2021-11-26T14:14:16.3730147Z 	at com.sonar.it.scanner.msbuild.TestUtils.runMSBuild(TestUtils.java:215)
2021-11-26T14:14:16.3731085Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:813)
2021-11-26T14:14:16.3732200Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:779)
2021-11-26T14:14:16.3733169Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:774)
2021-11-26T14:14:16.3734238Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.testProjectTypeDetectionWithWrongCasingReferenceName(ScannerMSBuildTest.java:699)

> 2021-11-26T14:14:16.7943569Z [ERROR]   ScannerMSBuildTest.testProjectTypeDetectionWithWrongCasingReferenceName:699->runBeginBuildAndEndForStandardProject:774->runBeginBuildAndEndForStandardProject:779->runBeginBuildAndEndForStandardProject:813 � Command
2021-11-26T14:14:16.7944904Z [INFO] 
2021-11-26T14:14:16.7945489Z [ERROR] Tests run: 34, Failures: 0, Errors: 1, Skipped: 6

### Known workarounds

Restart the failed job.

>2021-11-26T16:01:38.5945678Z [ERROR] Tests run: 34, Failures: 0, Errors: 1, Skipped: 6, Time elapsed: 591.542 s <<< FAILURE! - in com.sonar.it.scanner.SonarScannerTestSuite
2021-11-26T16:01:38.5947665Z [ERROR] testProjectTypeDetectionWithWrongCasingReferenceName(com.sonar.it.scanner.msbuild.ScannerMSBuildTest)  Time elapsed: 63.765 s  <<< ERROR!
2021-11-26T16:01:38.5948912Z com.sonar.orchestrator.util.CommandException: Timeout exceeded: 60000 ms [command: cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe" "/t:Restore,Rebuild" DotnetProjectTypeDetection.sln"]
2021-11-26T16:01:38.5950454Z 	at com.sonar.orchestrator.util.CommandExecutor.execute(CommandExecutor.java:98)
2021-11-26T16:01:38.5952977Z 	at com.sonar.it.scanner.msbuild.TestUtils.runMSBuildQuietly(TestUtils.java:263)
2021-11-26T16:01:38.5960231Z 	at com.sonar.it.scanner.msbuild.TestUtils.runMSBuild(TestUtils.java:215)
2021-11-26T16:01:38.5961176Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:813)
2021-11-26T16:01:38.5962139Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:779)
2021-11-26T16:01:38.5962977Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.runBeginBuildAndEndForStandardProject(ScannerMSBuildTest.java:774)
2021-11-26T16:01:38.5963853Z 	at com.sonar.it.scanner.msbuild.ScannerMSBuildTest.testProjectTypeDetectionWithWrongCasingReferenceName(ScannerMSBuildTest.java:699)

This issue is not specific to VS2017 image. It just happened on VS2019. I've updated the title.
```
[ERROR] Tests run: 34, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 767.255 s <<< FAILURE! - in com.sonar.it.scanner.SonarScannerTestSuite
[ERROR] testProjectTypeDetectionWithWrongCasingReferenceName(com.sonar.it.scanner.msbuild.ScannerMSBuildTest)  Time elapsed: 85.195 s  <<< ERROR!
com.sonar.orchestrator.util.CommandException: Timeout exceeded: 60000 ms [command: cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe" "/t:Restore,Rebuild" DotnetProjectTypeDetection.sln"]
```

Ok, I saw on a PR a different test failing now:

```
[ERROR] Tests run: 34, Failures: 0, Errors: 1, Skipped: 2, Time elapsed: 901.283 s <<< FAILURE! - in com.sonar.it.scanner.SonarScannerTestSuite
[ERROR] testNoProjectNameAndVersion(com.sonar.it.scanner.msbuild.ScannerMSBuildTest)  Time elapsed: 63.527 s  <<< ERROR!
com.sonar.orchestrator.util.CommandException: Timeout exceeded: 60000 ms [command: cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe" /t:Rebuild"]
	at com.sonar.orchestrator.util.CommandExecutor.execute(CommandExecutor.java:98)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at com.sonar.orchestrator.util.CommandExecutor.execute(CommandExecutor.java:89)
	... 43 more

```

I am not sure if we should disable it as well ... or change the timeout? However I don't understand why such a test would take more than 60 seconds!

--- a/its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java
+++ b/its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java
@@ -270,6 +270,7 @@ public class TestUtils {
     boolean mustRetry = true;
     while (mustRetry && attempts < MSBUILD_RETRY) {
       status = CommandExecutor.create().execute(Command.create(msBuildPath.toString())
+        .addArguments("-nodeReuse:false")
         .addArguments(arguments)
         .setDirectory(projectDir.toFile()), writer, 60 * 1000);
 

